services/frontend-v3/src/components/searchBar/SearchBarView.jsx

```diff
@@ -20,17 +29,29 @@ function SearchBarView(props) {
     handleSearch(values);
   };
 
+  const handleKeyPress = e => {
+    console.log("toggled1");
+    if (e.charCode === 32 || e.charCode === 13) {
+      e.preventDefault();
+      console.log("toggled2");
+      toggle();
+    }
+  };
+
   return (
-    <Form
-      form={form}
-      onFinish={onFinish}
-      style={{
-        width: "100%",
-        paddingLeft: "50px",
-        paddingRight: "50px",
-        paddingTop: "60px"
-      }}
-    >
+    <Form form={form} onFinish={onFinish} style={styles.outerForm}>
+      {empty === true ? (
+        <Alert
+          message={props.intl.formatMessage({
+            id: "alert.empty.search",
+            defaultMessage: "Please input a value into the search bar below"
+          })}
+          type="error"
+          style={styles.alert}
+        />
+      ) : (
+        setDoNothing[true]
```

Comment:
That's not how it should work and you probably do get the warning. if you want, you can do temp = useState... and then do setDoNothing=temp[1]

Reply:
wait why should my setDoNothing not be allowed to be set to true? If i create a temp state using useState then am I not going to need setTemp?? I think im missing something here...

Reply:
You aren't setting here. First of all, you are using square brackets to call a function. Secondly, setDoNothing was not a function before. When you use useState, the names of the variables don't matter, the order does since it is an array, not an object. And the method I suggested above is to remove the error associated with what you're doing. It is better now since you added "doNothing" to the useState but you'll get the error for not using in and you still aren't calling "setDoNothing" and a function.